StarryExpanse/EverPresent/StrangerController.cpp

```diff
@@ -93,13 +93,44 @@ void AStrangerController::AddHorizontalMouseScan(float amount) {
   if (!IsCursorLockedToCenter) {
     HorizontalMousePosition =
         FMath::Clamp(HorizontalMousePosition + amount, 0.0f, 1.0f);
+
+    // check to see if we're within +tolerance for screen border
+    if (1 - HorizontalMousePosition < HorizontalMouseNudgeThreshold) 
+	{
+		//rotate the camera in accordance with how close the cursor is to the edge of the screen
+      this->AddYawInput(HorizontalMouseNudgeThreshold -
+                        (1 - HorizontalMousePosition));
+    }
+
+	// check to see if we're within -tolerance for screen border
+	if (HorizontalMousePosition < HorizontalMouseNudgeThreshold)
+	{
+      // rotate the camera in accordance with how close the cursor is to the edge of the screen
+      this->AddYawInput(
+              -(HorizontalMouseNudgeThreshold - HorizontalMousePosition));
+	}
   }
 }
 
 void AStrangerController::AddVerticalMouseScan(float amount) {
   if (!IsCursorLockedToCenter) {
     VerticalMousePosition =
         FMath::Clamp(VerticalMousePosition + amount, 0.0f, 1.0f);
+
+    // check to see if we're within +tolerance for screen border
+    if (1 - VerticalMousePosition < VerticalMouseNudgeThreshold) 
+	{
+      // rotate the camera in accordance with how close the cursor is to the edge of the screen
+      this->AddPitchInput(
+          VerticalMouseNudgeThreshold - (1 - VerticalMousePosition));
+    }
+
+	// check to see if we're within -tolerance for screen border
```

Comment:
Indentation for this is off, it makes it look like this is within the previous if statement. You can run `./fix-formatting.sh` (https://github.com/starryexpanse/StarryExpanse/blob/master/fix-formatting.sh) to make the formatting easier.

Reply:
had VS unify the formatting.